community-membership.md

```diff
@@ -296,5 +297,69 @@ with the project. When a contributor returns to being more active, they may be
 promoted back to their previous position at the discretion of the current
 maintainers following the process defined above.
 
+## Specification Sponsor
+
+Specification sponsors are trusted collaborators of the technical committee, and
+work to review, approve, and sponsor [opentelemetry-specification][] issues and
+PRs.
+
+Members are defined
+in [Community Members -> Specification and Protos](./community-members.md#specifications-and-proto)
+and are members of the `spec-sponsors` team (TODO: add link to github team when
+available). Specification sponsors SHOULD list
+their [areas of interest](./areas-of-interest.md).
+
+### Requirements
+
+Specification sponsors are trusted to drive specification work which can be
+complex and which highly impactful to downstream language SIGs and to the
+OpenTelemetry project as a whole. Approvals from specification sponsors count
+towards the approvals required to merge specification PRs. Therefore, there is a
+high bar to becoming a specification sponsor.
+
+- Contributor to OpenTelemetry project for at least 1 year
```

Comment:
Thinking about emerging areas (Profiling?) and areas where we don't have much representation (mobile/browser instrumentation), would it make sense to relax on this requirement on exceptional cases?

Reply:
That's an interesting question. Profiling hasn't made it to the spec yet, but according to [the meeting notes](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit#heading=h.2oh8ox7yooz4) the group has been meeting since June 2022. Do we formally define contributor anywhere? Is a "contribution" limited to PRs, reviews of issues / PRs, or does participating in a SIG count?

Reply:
@svrnm, we _just_ talked about this during today's triage session, and here's another situation where contribution isn't code. 

@jack-berg, we have this definition, which already supports non-code contributions, but @svrnm will make this more explicit in a future PR.

https://github.com/open-telemetry/community/blob/main/community-membership.md#member

Reply:
Left a comment over on #2080. Do you think we need to do anything additional on this PR, or is it implied that the same types of contributions that count towards community membership count towards spec sponsor?

Reply:
Nothing to change here, in my opinion.